presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+* Fix an issue where analyze properties would remain in the ``system.metadata.analyze_properties`` table even after the connector was removed.
+* Add :func:`ST_Length` for spherical geography
+* Add ``view_owner`` column to the ``information_schema.views`` system table.
+* Add a json version of the query plan to ``QueryCompletedEvent``.
+* Add support for creating warnings during parsing.
+* Add a warning for using reserved word ``current_role`` as an identifier.
+* Add support for using the empty string as a delimiter for the :func:`split` function.
+  When an empty string is used as the delimiter, the string will be split into individual characters.
+
+Raptor Changes
+--------------
+* Add ``raptor.minimum-node-count`` configuration property. The the number of nodes in a cluster is fewer then the configured amount, data recovery and reassignment will not run.
+* Advance the first run of bucket rebalancer. Now bucket balancer will run after a grace period (configured in `raptor.startup-grace-period`) from coordinator restart.
+
+Hive Connector Changes
+----------------------
+
+* Add `hive.ignore-table-bucketing` configuration property and `ignore_table_bucketing` session property.  When set to true these properties enable reading from partitions whose bucketing scheme does not match the table bucketing scheme.
+
+Verifier Changes
+----------------
+* Fix an issue where the determinism check for ``Insert`` queries was not run when there was a result mismatch.
+* Fix an issue where the checksum query was treated as the main query, and the main query was treated as part of the setup queries, for ``Insert`` query verification.
+* Fix query timeout enforcement by replacing local timer with session property ``query_max_execution_time``.
+* Improve result comparison for floating point columns by using relative errors. Replace configuration property ``expected-double-precision`` with ``relative-error-margin``.
+* Improve result comparison for orderable array columns by applying :func:`array_sort` before :func:`checksum`.
+* Improve result comparison for ``Select`` queries by rewriting ``Select`` queries as ``CreateTableAsSelect`` and using checksum queries to verify the results. This eliminates the row count limit for Select queries.
+* Improve Verifier interfaces to allow easy extension by community users.
+* Reuse initial control query results for the determinism check. This reduces the maximum number of control query runs from 3 to 2 and also eliminates test query reruns.
+* Add support for retrying transient Presto query failures, using configuration properties ``presto.max-attempts``, ``presto.min-backoff-delay``, ``presto.max-backoff-delay``, ``presto.backoff-scale-factor``. Similar configurations prefixed with ``cluster-connection`` instead of ``presto`` are introduced for retrying transient network failures when talking to Presto server. Intermediate failures are recorded and emitted in the output.
```

Comment:
`talking to Presto server.` -> `communicating with the Presto server`

Are we talking about the coordinator here? server is vague.

Reply:
This could either be a coordinator or a gateway. But since gateway is not an open-source concept, we can just say coordinator?

Reply:
Let's say `communicating with the Presto coordinator`.